tools/cli/persistenceUtil.go

```diff
@@ -83,19 +95,32 @@ func CreateDefaultDBConfig(c *cli.Context) (config.DataStore, error) {
 		}
 
 		if c.Bool(FlagEnableTLS) {
```

Comment:
[Nit] you don't need the if c.Bool check here anymore. You can just do:

defaultConfig.TLS = tls always

Reply:
or actually just assign it when initializing config

Reply:
done